### PR TITLE
Fixed version for 'Cortex XDR - Port Scan - Adjusted'

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7853

## Description
Improve the 'Cortex XDR - Port Scan - Adjusted' playbook to include proper analysis, containment, investigation, and remediation steps.

### CODE REVIEW

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml [hunk 1]
```diff
@@ -595,59 +531,1460 @@ tasks:
...
+      - "142"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: RelatedFiles
```
Comment (efelmandar) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml:834]: Where do we get this key from?

**Review by efelmandar — COMMENTED** [review]
Nice work! just a couple of small things

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml (hunk 1 above)
Comment (TalNos) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml:834]: these are context keys extracted by the 'TIM - Indicator Relationships Analysis' playbook. should I change the sub-playbooks context to 'Shared globally'?

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml [hunk 2]
```diff
@@ -595,59 +531,1460 @@ tasks:
...
+    type: condition
+    task:
+      id: 7df2f070-6394-4152-80ca-a485b961aaae
+      version: -1
+      name: Has hunt results?
+      description: Verifies whether there are any threat hunting results.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
```
[the remark this answers is not in the document]
Comment (TalNos) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml:1017]: done

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml (hunk 1 above)
Comment (efelmandar) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml:834]: No, if it's in the sub playbook outputs than it's ok as private context 👍

**Pull request merged**